lib/common/commonUtil.py

```diff
@@ -4,27 +4,26 @@
 import platform
 import subprocess
 import numpy as np
-from datetime import datetime
+import time
 from environment import Environment
 from logConfig import get_logger
-
+from datetime import datetime
 logger = get_logger(__name__)
 
 
 class StatusRecorder(object):
```

Comment:
It will be helpful if we add the comments here to describe the data structure of status JSON file.